Fix wide-attachment-image-controls-basic test

squelart opened the pull request:
#### 44fccba78690ca1160f1d8c8776103e8540a0bae
<pre>
Fix wide-attachment-image-controls-basic test
<a href="https://bugs.webkit.org/show_bug.cgi?id=259528">https://bugs.webkit.org/show_bug.cgi?id=259528</a>
<a href="https://rdar.apple.com/112918166">rdar://112918166</a>

Reviewed by Tim Nguyen.

Delay the rendering screenshot until the attachment icon has loaded.

* LayoutTests/fast/attachment/mac/wide-attachment-image-controls-basic.html:
* LayoutTests/platform/mac-wk2/TestExpectations:

* LayoutTests/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt:
* LayoutTests/platform/mac-monterey/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt: Renamed from LayoutTests/platform/mac/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt.
There is a slight difference between how Monterey and Sonoma blend
the attachment background, which is the tertiary-fill (0,0,0,5%).
5% of 255 is 12.75.
- On Monterey this is rounded up to 13 -&gt; #0000000D
- On Sonoma this is truncated down to 12 -&gt; #0000000C

Canonical link: <a href="https://commits.webkit.org/272548@main">https://commits.webkit.org/272548@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/284fab800b73d343c4135ad1d7f3a4409397b736

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/31962 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/10662 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/33719 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/34479 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/28960 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/13025 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/7899 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/28555 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/32335 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/9018 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/28568 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/7806 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/7985 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/28481 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/35823 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/29080 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/28935 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/34085 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/8070 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/6056 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/31944 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/9715 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/28288 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7487 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/8731 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/8593 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Not a fan of dump render tree tests in general, but I guess this is fine.

squelart committed:
Fix wide-attachment-image-controls-basic test
https://bugs.webkit.org/show_bug.cgi?id=259528
rdar://112918166

Reviewed by Tim Nguyen.

Delay the rendering screenshot until the attachment icon has loaded.

* LayoutTests/fast/attachment/mac/wide-attachment-image-controls-basic.html:
* LayoutTests/platform/mac-wk2/TestExpectations:

* LayoutTests/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt:
* LayoutTests/platform/mac-monterey/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt: Renamed from LayoutTests/platform/mac/fast/attachment/mac/wide-attachment-image-controls-basic-expected.txt.
There is a slight difference between how Monterey and Sonoma blend
the attachment background, which is the tertiary-fill (0,0,0,5%).
5% of 255 is 12.75.
- On Monterey this is rounded up to 13 -> #0000000D
- On Sonoma this is truncated down to 12 -> #0000000C

Canonical link: https://commits.webkit.org/272548@main
